src/redis.c

```diff
@@ -429,6 +464,8 @@ static int redis_read(void) /* {{{ */
     redis_handle_info(rn->name, rr->str, "total_bytes", "output",
                       "total_net_output_bytes", DS_TYPE_DERIVE);
 
+	redis_db_stats (rn->name, rr->str);
```

Comment:
Indentation is off here. Please run `clang-format -style=file -i src/redis.c` to fix these formatting issues.